src/chains-config/metadata-consts/karuraConsts.ts

```diff
@@ -0,0 +1,12 @@
+import { MetadataConsts } from '../../types/chains-config';
+import { extrinsicBaseWeight } from './substrateConsts';
+
+export const karuraDefinitions: MetadataConsts[] = [
+	{
+		runtimeVersions: [
+			1000, 1001, 1002, 1003, 1004, 1005, 1006, 1007, 1008, 1009, 1010, 1011,
```

Comment:
Do we need to list all the runtime versions? Do we need to PR here if we did a runtime upgrade?

Reply:
Yea and yes. If it doesnt have your new runtime version immediately thats alright, we just end up searching the decorated metadata for the weights for the missing runtime then cache it for the server instance.